poediscordbot/cogs/pob/output/aggregators/config_aggregator.py

```diff
@@ -46,8 +48,35 @@ def get_config_string(config):
                              bool(config.get(ifOption).get('value'))):
                 continue
 
+            if_skill = entry.get('ifSkill')
+            if_skill_list = entry.get('ifSkillList', [])
+            if if_skill:
+                if_skill_list.append(if_skill)
+
             config_line = ConfigAggregator.prepare_config_line(abbrev if abbrev else key, value)
             if category and config_line:
+                # Check if we have any skill active preconditions and try to
+                # match at least one of active gems with it
+                if if_skill_list:
```

Comment:
I'd kinda like to separate this logic by pulling out the breaks and continues, move it into a new function that just returns a boolean telling the loop to either skip or continue. 

That'd make it easier to read. If you cant I can suggest something a bit later on

Reply:
Yeah, it was simply change, updated